plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoTypeMapping.java

```diff
@@ -190,6 +190,8 @@ public void testDecimal()
                 .addRoundTrip("NumberDecimal(\"2\")", "CAST('2' AS decimal(1, 0))")
                 .addRoundTrip("NumberDecimal(\"2.3\")", "CAST('2.3' AS decimal(2, 1))")
                 .addRoundTrip("NumberDecimal(\"-2.3\")", "CAST('-2.3' AS decimal(2, 1))")
```

Comment:
We had no tests for values < 1 which seems to match the cause.

It means that even though we were able to infer precision and scale for `0.x` values the inferred values were wrong. Do I understand correctly?

Reply:
Yes, but more specifically for values <0.1, so with leading zeros